west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: f3f276dd2ecad981a2abf1d87c3d3d0b2e9d5373
+      revision: 7735bc6d171ab2fe6590ab47171f5decdbe2ddfd
```

Comment:
Please update to
e4827fba2254402b99a7be249974de4e0acbcba1
https://github.com/nrfconnect/sdk-zephyr/commit/e4827fba2254402b99a7be249974de4e0acbcba1

Reply:
Done